app/services/coronavirus_pages/timeline_entry_builder.rb

```diff
@@ -0,0 +1,29 @@
+class CoronavirusPages::TimelineEntryBuilder
+  def create_timeline_entries
+    if timeline_entries_from_yaml.any?
```

Comment:
Minor it'd be a more idiomatic Ruby to have this as a guard clause. E.g. `return if timeline_entries_from_yaml.empty?`

Do you definitely want to abort if there are no entries though or should it delete all items? I'm not sure why we're anticipate this being empty to know what we need to be cautious of.

Reply:
This was also just copied from some [work that Leena already did](https://github.com/alphagov/collections-publisher/blob/e47d09b50f80942e9db7eb30200c5fffe5524d0e/app/services/coronavirus_pages/announcements_builder.rb#L3). I think it's just to protect us from losing data if someone removes all the entries from the yaml, Leena might know a bit more as to why this is desirable. I've changed the syntax of the guard clause.